Atmosphere/style.css

```diff
@@ -0,0 +1,129 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  background-position: center center;
+  background-size: cover;
+  color: #fff;
+}
+
+.container {
+  padding: 16em 3em;
+  text-align: center;
+  background-image: linear-gradient(
+      to top,
+      rgba(71, 46, 104, 0.9),
+      rgba(71, 46, 104, 0.9)
+    ),
+    url(images/banner.jpg);
+}
+.container h4 {
+  font-size: 20px;
+}
+.container h1 {
+  font-size: 36px;
+}
+.btn {
+  padding: 12px 34.2px;
+  border: 1px solid #00a6d9;
+  width: 80%;
+  height: 30px;
+  background-color: #00a6d9;
+  display: inline-block;
+  text-decoration: none;
+  background-color: #00a6d9;
+  color: #ffffff;
+  text-transform: uppercase;
+}
+.container-card {
+  background-color: #f5f8fa;
+}
+.card {
+  padding: 25px;
+  margin: 25px;
+  background-color: #fff;
+}
+.card h4 {
+  color: #00a6d9;
+  text-transform: uppercase;
+}
+.card h2 {
+  color: #637485;
+  font-size: 24px;
+}
+.card p {
+  color: #637485;
+  margin-top: 60px;
+}
+.btn-one {
+  font-size: 11px;
+  font-weight: bold;
+  width: 80%;
+  margin-top: 15px;
+  padding: 14px;
+  border: 1px solid #dde1e3;
+  text-transform: uppercase;
+  background-color: transparent;
+  border-radius: 5px;
+  color: #7b8a98;
+  display: inline-block;
```

Comment:
display should be block in this case if you debug you will get to know
use margin: 10px auto; to align the items in the center
using margin-top and margin-bottom will only add margin to top and bottom what about let and right ?